test/e2e/invalid_resource.go

```diff
@@ -227,8 +227,7 @@ func errorExistsInLog(logFolder string, expectedError string) (bool, error) {
 			logLines := strings.Split(string(log), "\n")
 			for _, line := range logLines {
 				if strings.Contains(line, expectedError) &&
-					strings.Contains(line, clusterResources.Cluster.Namespace) &&
-					strings.Contains(line, clusterResources.Cluster.Name) {
+					strings.Contains(line, clusterResources.Cluster.Namespace) {
```

Comment:
Please edit the comment for this function, as it still states that it its confirming cluster name (line 215).  Consider adding to the comment to indicate why we can't check for name.